### DISCUSSION TITLE
502 error when attempt to export app

### DISCUSSION THREAD

**Discussion by Bong1hong** [Budibase/budibase#10159] [Help]
Hi,

I have encountered an 502 Bad Gateway error when attempting to export app.

Description:
First, it run for 3 - 7 minutes (load in browser tab).
Then, it stop and return this error.
![Budibase 502 Error](https://user-images.githubusercontent.com/18653633/228770101-f2c04d00-ccd0-4d30-a959-48b6a069e02f.png)

No additional messages (info / error) in console.
Thanks for any helps.

Additional Information:
Hosting Method: Self-hosted
Budibase version: 2.1.28
Browser: Google Chrome

**Comment by melohagan** [comment]
Hey @Bong1hong 

I have a distant memory of this being a bug. Please update to the latest version of Budibase!
